5_databases/week3/additional_queries.sql

```diff
@@ -0,0 +1,55 @@
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT *
+FROM meal
+WHERE price < 30;
+
+
+-- Get meals that still has available reservations
+SELECT meal.id, meal.title, meal.max_reservations, COUNT(reservation.id) AS reservations_taken
+FROM meal
+LEFT JOIN reservation ON meal.id = reservation.meal_id
+GROUP BY meal.id
+HAVING meal.max_reservations > COUNT(reservation.id) OR COUNT(reservation.id) IS NULL;
```

Comment:
Very nice approach, I didn't even know about `HAVING` 👍 

There is one issue with the query. Don't you think we should take into account how many people each reservation is made for? How would you do that?